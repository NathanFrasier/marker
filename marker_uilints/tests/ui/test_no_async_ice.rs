use std::path::Path;
use std::path::PathBuf;

pub async fn canonicalize(path: impl AsRef<Path>) -> std::io::Result<PathBuf> {
    let _path = path.as_ref().to_owned();
    Ok(PathBuf::from("./duck"))
}

fn main() {}
