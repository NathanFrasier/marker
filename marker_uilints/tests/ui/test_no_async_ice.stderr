warning: async blocks and await expressions are currently not supported
 --> $DIR/test_no_async_ice.rs:4:79
  |
4 |   pub async fn canonicalize(path: impl AsRef<Path>) -> std::io::Result<PathBuf> {
  |  _______________________________________________________________________________^
5 | |     let _path = path.as_ref().to_owned();
6 | |     Ok(PathBuf::from("./duck"))
7 | | }
  | |_^
  |
  = note: see rust-marker/marker#174

warning: 1 warning emitted

